content/doc/administration/requirements/jenkins-on-java-11.adoc

```diff
@@ -1,46 +1,4 @@
 ---
-layout: documentation
-title: Running Jenkins on Java 11
----
-
-If you are upgrading the Jenkins JVM version from Java 8 to Java 11, please link:/doc/administration/requirements/upgrade-java-guidelines[follow these guidelines].
-
-For simplicity, this document describes how to run the most recent version of Jenkins on Java 11.
-
-== Running Jenkins with Docker
-
-The easiest way to run Jenkins on Java 11 is with a Docker image.
-
-To use the latest weekly release of the Java 11-based images, use the `jdk11` tag. For other releases, append the `-jdk11` tag to the version (for example, `2.284-jdk11`).
-
-For example, this command starts Jenkins on Java 11 using the latest weekly release: 
-
-[source, shell]
-----
-docker pull jenkins/jenkins:jdk11
-docker run --rm -ti \
-  -p 8080:8080 -p 50000:50000 \
-  -v jenkins-home:/var/jenkins_home \
-  jenkins/jenkins:jdk11
-----
-
-== Running Jenkins with Java
-
-As with Java 8, you can start Jenkins using the `java` command:
-
-[source, shell]
-----
-java -jar jenkins.war
-----
-
-== Discovering issues with Java 11
-
-Between June 2018 and February 2019, the community performed many exploratory tests to discover as many Java 11 issues as possible.
-
-As a result, the community solved a lot of problems before announcing Java 11 support in Jenkins. However, it's still possible that some plugins haven't been updated to support Java 11.
-
-Compatibility issues are being tracked in the Jenkins issue tracker as link:https://issues.jenkins.io/issues/?jql=labels%20%3D%20java11-compatibility%20and%20status%20not%20in%20(Closed%2CResolved)[Known Java 11 Compatibility Issues].
-
-If you discover a Java 11 incompatibility, please link:/participate/report-issue/[report the issue] in the Jenkins bug tracker using the link:https://issues.jenkins.io/issues/?jql=labels%20%3D%20java11-compatibility%20and%20status%20not%20in%20(Closed%2CResolved)[`java11-compatibility` label].
-
-For security issues, please use the standard link:/security/#reporting-vulnerabilities[vulnerability reporting process].
+layout: redirect
+redirect_url: /doc/book/installing/
```

Comment:
This is a different method of creating a redirect than I am familiar with for our docs. It may be that this is how it is done for Jenkins. For docs.cloudbees.com we use :page-aliases: <old-page-title.adoc>.